packages/cardano-services/src/util/hexString.ts

```diff
@@ -0,0 +1 @@
+export const hexStringToBuffer = (hex: string) => Buffer.from(hex, 'hex');
```

Comment:
Also generic, but also probably not worth moving until we have implemented the util package.

Reply:
Yep, similar to [bytesToHex](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/core/src/util/misc/bytesToHex.ts) that's currently in `core`